src/internal/NoCodesInternal.ts

```diff
@@ -0,0 +1,31 @@
+import {NativeEventEmitter, NativeModules} from "react-native";
+import NoCodesApi from "../NoCodesApi";
+import NoCodesConfig from "../NoCodesConfig";
+import Mapper, { QNoCodesResult } from "./Mapper";
+
+const {RNNoCodes} = NativeModules;
+
+const sdkVersion = "1.0.0";
+
+const EVENT_NOCODES = "NoCodesEvent";
```

Comment:
Where is the logic of handling delegate events? I see removals of the `ActionResult` class, but nothing added instead, while the same should exist for No-Codes